src/components/providers/ProviderView.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { Box, Stack, Avatar, Text, useTheme, Badge } from "@chakra-ui/react";
+import { useNavigate } from "react-router-dom";
+
+const ProviderView = ({ id, name, title, bio, availability, image }) => {
+  const theme = useTheme();
+  const navigate = useNavigate();
+
+  return (
+    <Box
+      bgColor={theme.colors.neutral[0]}
+      _hover={{
+        bgColor: theme.colors.neutral[300],
+        cursor: "pointer",
+      }}
+      w="full"
+      rounded="md"
+      py={3}
+      px={6}
+      overflow="hidden"
+      maxH="167px"
+      onClick={() => {
+        navigate(`/provider/profile/${id}`);
+      }}
+    >
+      <Stack direction={"row"} spacing={4} align={"center"}>
+        <Avatar
```

Comment:
Nit: The designs have a border on these avatar with a small space between the border and the image
![CleanShot 2022-04-13 at 17 10 59](https://user-images.githubusercontent.com/19275035/163271013-fe26c754-1e7e-4982-bb0c-9326c44dba0d.png)
.